server/src/internalClusterTest/java/org/opensearch/indices/store/IndicesStoreIntegrationIT.java

```diff
@@ -168,6 +168,97 @@ public void testIndexCleanup() throws Exception {
+     * and dont wait for second relocation to complete.
+     */
+    public void testIndexCleanupWhenMultipleRelocation() throws Exception {
+        internalCluster().startNode(nonDataNode());
+        final String node_1 = internalCluster().startNode(nonMasterNode());
+        final String node_2 = internalCluster().startNode(nonMasterNode());
+        logger.info("--> creating index [test] with one shard and on replica");
+        assertAcked(prepareCreate("test").setSettings(
+            Settings.builder().put(indexSettings())
+                .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
+                .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 1))
+        );
+        ensureGreen("test");
+        ClusterState state = client().admin().cluster().prepareState().get().getState();
+        Index index = state.metadata().index("test").getIndex();
+
+        logger.info("--> making sure that shard and its replica are allocated on node_1 and node_2");
+        assertThat(Files.exists(shardDirectory(node_1, index, 0)), equalTo(true));
+        assertThat(Files.exists(indexDirectory(node_1, index)), equalTo(true));
+        assertThat(Files.exists(shardDirectory(node_2, index, 0)), equalTo(true));
+        assertThat(Files.exists(indexDirectory(node_2, index)), equalTo(true));
+
+        logger.info("--> starting node server3");
+        final String node_3 = internalCluster().startNode(nonMasterNode());
+        logger.info("--> starting node server4");
+        final String node_4 = internalCluster().startNode(nonMasterNode());
+
+        ClusterHealthResponse clusterHealth = client().admin().cluster().prepareHealth()
+            .setWaitForNodes("5")
+            .setWaitForNoRelocatingShards(true)
+            .get();
+        assertThat(clusterHealth.isTimedOut(), equalTo(false));
+
+        assertThat(Files.exists(shardDirectory(node_1, index, 0)), equalTo(true));
+        assertThat(Files.exists(indexDirectory(node_1, index)), equalTo(true));
+        assertThat(Files.exists(shardDirectory(node_2, index, 0)), equalTo(true));
+        assertThat(Files.exists(indexDirectory(node_2, index)), equalTo(true));
+        assertThat(Files.exists(shardDirectory(node_3, index, 0)), equalTo(false));
+        assertThat(Files.exists(indexDirectory(node_3, index)), equalTo(false));
+        assertThat(Files.exists(shardDirectory(node_4, index, 0)), equalTo(false));
+        assertThat(Files.exists(indexDirectory(node_4, index)), equalTo(false));
+
+        MockTransportService transportService = (MockTransportService) internalCluster().getInstance(TransportService.class, node_4);
+
+        transportService.addMessageListener(new TransportMessageListener() {
+            @Override
+            public void onRequestReceived(long requestId, String action) {
+                if (action.equals(PeerRecoveryTargetService.Actions.FILE_CHUNK)) {
+                    try{
```

Comment:
checkstyle: space after try

Reply:
Somehow didn't captured in checkstyle test in precommit. 

Corrected.